apps-api/backoffice/src/modules/admin/election/services.ts

```diff
@@ -467,6 +471,72 @@ export class AdminElectionService {
     return ok()
   }
 
+  async uploadElectionOnsiteResult(
+    electionId: string,
+    result: { candidateId: string; votes: number }[]
+  ) {
+    const electionResult = await this.adminElectionRepository.getElectionById(electionId)
+    if (electionResult.isErr()) {
+      return mapRepositoryError(electionResult.error, {
+        RECORD_NOT_FOUND: {
+          code: InternalErrorCode.ELECTION_NOT_FOUND,
+          message: `Cannot found election id ${electionId}`,
+        },
+      })
+    }
+
+    const election = electionResult.value
+    if (election.type !== 'ONSITE' && election.type !== 'HYBRID') {
+      return err({
+        code: InternalErrorCode.ELECTION_INVALID_TYPE,
+        message: `Cannot upload onsite result for election type ${election.type}`,
+      })
+    }
+
+    if (election.isCancelled) {
+      return err({
+        code: InternalErrorCode.ELECTION_IS_CANNCELLED,
```

Comment:
Please fix this after merging #198 
```suggestion
        code: InternalErrorCode.ELECTION_IS_CANCELLED,
```